scheduler/Dockerfile.dataflow

```diff
@@ -14,7 +14,7 @@ RUN gradle build --no-daemon --info
 ################################################################################
 
 # Some dependencies require glibc, which Alpine does not provide
-FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.14-2
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.14-2.1681917142
```

Comment:
:question: Would `latest` be preferable to the specific version?

Reply:
We have not concluded yet whether it is alright to use `latest` tag. So bumping up the version for now.

Reply:
`1.14-2` is not a rolling tag (`1.14` is).

Reply:
I'd probably stick to `1.14` then same way as I'd use for base `ubuntu:22.04` and not specific build of it - but you are right we did not make specific decision on these yet.